src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -92,9 +92,12 @@
     @Option(shortName = "Q", doc = "Minimum base quality for a base to contribute coverage.", overridable = true)
     public int MINIMUM_BASE_QUALITY = 20;
 
-    @Option(shortName = "CAP", doc = "Treat bases with coverage exceeding this value as if they had coverage at this value.", overridable = true)
+    @Option(shortName = "CAP", doc = "Treat positions with coverage exceeding this value as if they had coverage at this value (but calculate the difference for PCT_EXC_CAPPED).", overridable = true)
     public int COVERAGE_CAP = 250;
 
+    @Option(shortName="LOCUS_ACCUMULATION_CAP", doc="At positions with coverage exceeding this value, completely ignore reads that accumulate beyond this value (so that they will not be considered for PCT_EXC_CAPPED).  Used to keep memory consumption in check, but could create bias if set too low")
+    public int LOCUS_ACCUMULATION_CAP = 100000;
```

Comment:
check that LOCUS_ACCUMULATION_CAP is bigger than COVERAGE_CAP. I woudl actualyl recommend that it be much bigger, say 5x times so that the exclusion actually does something.